invenio_previewer/version.py

```diff
@@ -14,4 +14,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = "1.0.0a11"
+__version__ = "1.0.0a12"
```

Comment:
I would revert this for now, no? We will do it later, there is also the PR with the doc.